samples/extensions/shader_object/shader_object.cpp

```diff
@@ -1,6 +1,11 @@
 /*
+<<<<<<< HEAD
+ * Copyright 2023-2024 Nintendo
+ * Copyright 2023-2024, Sascha Willems
+=======
  * Copyright 2023-2024-2024 Nintendo
  * Copyright 2023-2024-2024, Sascha Willems
+>>>>>>> main
```

Comment:
Looks like there are merge conflict markers left in this file

Reply:
Sorry, my bad. Didn't realize this and it compiled fine due to being in a comment. Should be fixed by now :)